### ISSUE TITLE
Xyce simulation does not work with IHP

### ISSUE THREAD

**Issue by hpretl** [iic-jku/IIC-OSIC-TOOLS#86]
Running the stock IHP simulation example, the following pops up:

<img width="701" alt="image" src="https://github.com/user-attachments/assets/a6105c9c-6061-4bf4-962d-ca19a82f34f2" />

**Comment by Abdulrahman-AL-Sabagh** [comment]
Hello
Regarding this problem
I found that a xyce plugin called ADMS should be installed in order to fix this error
There is also a necessary instruction that should be added to the XYCE configurations
![image](https://github.com/user-attachments/assets/ab746ddf-41e0-482d-850b-ec7377bf01f9)
the reference for it in the docs https://github.com/Xyce/Xyce/blob/master/INSTALL.md#adding-the-xyceadms-verilog-a-model-compiler

**Comment by hpretl** [comment]
Yes, thanks for the hint! I have meanwhile identified and resolved the issue. The next release will have this plugin for Xyce properly installed.
